src/stavenote.ts

```diff
@@ -370,10 +368,10 @@ export class StaveNote extends StemmableNote {
   constructor(noteStruct: StaveNoteStruct) {
     super(noteStruct);
     this.setAttribute('type', 'StaveNote');
-    // Ledger Lines default width 2.0
-    this.ledgerLineStyle = { lineWidth: 2.0 };
-    this.clef = noteStruct.clef;
-    this.octave_shift = noteStruct.octave_shift;
+
+    this.ledgerLineStyle = { lineWidth: 2.0 }; // Set default width of ledger lines to 2.0.
```

Comment:
Comments are on the line above in the `coding template`: accidental.ts according to the wiki.

Reply:
Sounds good! I'll make sure that implementation comments precede the line of code.